app/services/integrations/aggregator/taxes/invoices/void_service.rb

```diff
@@ -31,11 +31,20 @@ def call
           private
 
           def payload
-            [
-              {
-                "id" => invoice.id
-              }
-            ]
+            if integration.type.to_s == "Integrations::AnrokIntegration"
+              [
+                {
+                  "id" => invoice.id
+                }
+              ]
+            else
+              [
+                {
+                  "company_code" => integration.company_code,
+                  "id" => invoice.id
+                }
+              ]
+            end
```

Comment:
is this a general case of does it only applies to Avalara? if only applies to Avalara I would use a case statement instead of if/else otherwise it might require more work adding a 3rd aggregator integration.